src/dg/dg.h

```diff
@@ -510,7 +525,14 @@ class DGBase
     /// High order grid that will provide the MappingFEField
     std::shared_ptr<HighOrderGrid<dim,real,MeshType>> high_order_grid;
 
+    /// Operators base that will provide the Operators
+    OPERATOR::OperatorBase<dim,real> operators;
+    /// Sets the current time within DG to be used for unsteady source terms.
+    void set_current_time(const real current_time);
+
 protected:
+    ///The current time for explicit solves
+    real current_time;
```

Comment:
could access `current_time` from `ODESolverBase` directly instead

Reply:
Agreed. DG is a *spatial* discretization. DG should not have a `time` state. Having the behavior of the class depend on a `time` state is not something you would expect out of a spatial discretization.

Reply:
The issue is that for DG to use current_time from ODESolverBase, then DG would need to store an ODESolver object. I agree that the spatial discretization shouldn't have time, but it most definitely shouldn't have an ODESolver object since the ODESolver is separate from the spatial discretization. Thus, in order to have the source term be time-dependent, which is needed for the unsteady problems, either DG needs to store the current_time or it needs the ODESolver object. I went with the former. If you have a suggestion on how to use ODESolverBases's current_time without DG needing info about the object I will gladly change it.

Reply:
Leaving this as fixed as per my explanation above.

Reply:
We just need to figure out a way to have time dependant source terms in `PhysicsBase`; this would be relevant for any flow that has some sort of forcing function. Or consider moving this kind of source term out of physics and have it be added outside somewhere that knows about time since it's flow case specific / something additional to the underlying physics equations. Maybe this would fall under the `PhysicsModel` class in pull-request #76 ? Either way, DG would have to know about time if ODE cannot currently communicate with the `PhysicsBase` class

Reply:
As we discussed yesterday I will resolve this in a future PR.

Reply:
Resolving this converstation as it will be addressed by issue #186